src/olympia/bandwagon/templates/bandwagon/ajax_new.html

```diff
@@ -6,7 +6,7 @@
       {{ field(form.name, _('Name:')) }}
       {% set slug_base = url('collections.user', user.username, add_prefix=False) %}
       <p id="collection-form-slug">
-        {{ form.errors['slug'] }}
+        {% if form.errors.slug %}{{ form.errors.slug }}{% endif %}
```

Comment:
Just cleaning up along the way, it's one of the undefined errors printed out by our new jinja config (only locally though, still messy and should never happen)